.github/workflows/build-and-push.yml

```
@@ -0,0 +1,26 @@
+name: Build to release branches
+
+on:
+  workflow_dispatch:
+  push:
+    tags: ['*']
+    branches:
+      - dev-beta
+      - dev-main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  build-assets:
+    uses: inpsyde/reusable-workflows/.github/workflows/build-and-push-assets.yml@main
+    with:
+      PHP_VERSION: '8.1'
```

As per https://github.com/RRZE-Webteam/FAU-Studium/pull/125#discussion_r1584314349.
```suggestion
      PHP_VERSION: '8.2'
```

---

Ah, sorry I think I have missed that after comment update in the other PR and it's 8.1 in there too, but If production is on `8.1`, won't it be dangerous to use `8.2` here? because `composer install` might install a package with minimum PHP 8.2 requirement and cause fatal in the released plugin.

---

>and it's 8.1 in there too

Sorry, I have no idea how I overlooked that. Probably because I only checked my suggestion and not your actual change. What is wrong with me? Sorry for the noise! 😬